page/about-jquery/using-jquery.md

```diff
@@ -0,0 +1,155 @@
+### jQuery Fundamentals
+
+This tutorial will help you try out some common uses of jQuery.
+
+If you haven't setup a demo HTML page that includes a copy of jQuery, see [How To Setup jQuery](http://learn.jquery.com/about-jquery/how-to-setup-jQuery/) for the complete tutorial on how to get started, or simply follow along with the examples below.
+
+### Adding and Removing an HTML Class
+
+jQuery is often used to bind events to elements in the DOM, assigning them functionality to be triggered by a user's actions on the page. To demonstrate, let's highlight the menu item a user last clicked by changing its HTML class to "active", and assigning the "active" class a different appearance using CSS.
+
+First, copy or create the `index.html` file below:
```

Comment:
Hrm... Maybe we could link to a codepen / jsbin / etc?  Do we have a policy for this?